R/zzz.R

```diff
@@ -72,6 +73,20 @@ R7_union <- new_class(
   }
 )
 
+#' @export
+str.R7_union <- function(object, ..., nest.lev = 0) {
+  cat(if (nest.lev > 0) " ")
+  cat("<R7_union>: ", class_desc(object), sep = "")
+  cat("\n")
+
+  if (nest.lev == 0) {
+    props <- props(object)
+    props$srcref <- NULL
```

Comment:
Where does this come from?

Reply:
No idea; and it's not needed. I'll remove.